apps/penxle.com/src/routes/editor/GalleryEditor.svelte

```diff
@@ -368,7 +370,7 @@
       {/each}
     {/if}
 
-    {#if content?.content && content.content.length > 1}
+    {#if content?.content && (content.content.length > 1 || (content.content.length === 1 && content.content[0].type !== 'paragraph'))}
```

Comment:
P2: filter 한 다음 배열 길이를 비교하면 더 간결하게 할 수 있습니다.

```suggestion
    {#if content?.content.filter(({ type }) => type === 'image').length > 0 }
```